Xamarin.Forms.Platform.Android/FastRenderers/AccessibilityProvider.cs

```diff
@@ -145,8 +145,7 @@ bool SetHint(string hint = GetFromElement)
 			string value = hint;
 			if (value == GetFromElement)
 			{
-				value = string.Join(". ", (string)Element.GetValue(Accessibility.NameProperty),
-					(string)Element.GetValue(Accessibility.HintProperty));
+				value = string.Join((String.IsNullOrWhiteSpace((string)(Element.GetValue(Accessibility.NameProperty))) || String.IsNullOrWhiteSpace((string)(Element.GetValue(Accessibility.HintProperty)))) ? "" : ". ", (string)Element.GetValue(Accessibility.NameProperty), (string)Element.GetValue(Accessibility.HintProperty));
```

Comment:
by not understanding the meaning of GetFromElement, i would do something like this
`bool SetHint()
		{
			if (Element == null || Control == null)
			{
				return false;
			}

			var textView = Control as TextView;
			if (textView == null)
			{
				return false;
			}

			// Let the specified Title/Placeholder take precedence, but don't set the ContentDescription (won't work anyway)
			if (((Element as Picker)?.Title ?? (Element as Entry)?.Placeholder) != null)
			{
				return true;
			}

			string accessibilityName = (string)Element.GetValue(Accessibility.NameProperty);
			string accessibilityHint = (string)Element.GetValue(Accessibility.HintProperty);

			if (string.IsNullOrWhiteSpace(accessibilityName) && string.IsNullOrWhiteSpace(accessibilityHint))
			{
				return true;
			}

			textView.Hint = string.Join(". ", accessibilityName, accessibilityHint);

			return true;
		}`